src/main/groovy/nextflow/validation/config/HelpConfig.groovy

```diff
@@ -3,23 +3,56 @@ package nextflow.validation.config
 import groovy.util.logging.Slf4j
 
 import static nextflow.validation.utils.Colors.removeColors
+
+import nextflow.config.schema.ConfigOption
+import nextflow.config.schema.ConfigScope
+import nextflow.config.schema.ScopeName
+import nextflow.script.dsl.Description
+
 /**
- * This class allows to model a specific configuration, extracting values from a map and converting
+ * This class is used to read, parse and validate the `validation.help` config block.
  *
  * @author : nvnieuwk <nicolas.vannieuwkerke@ugent.be>
  *
  */
 
 @Slf4j
+@ScopeName('validation.help')
+@Description('''
+    The `validation.help` scope allows you to configure the help message produced by the `nf-schema` plugin.
+''')
```

Comment:
`@Description` is not needed for nested config scope classes because you already defined it for `ValidationConfig::help`